
From: Ingo Molnar <mingo@elte.hu>

found one missing bit while double-checking 2.6.7-mm1: this chunk was in
the full patch i sent but not in the delta patch and you added the delta
as nx-update.patch. So another delta is attached.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/i386/mm/init.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN arch/i386/mm/init.c~nx-update-2 arch/i386/mm/init.c
--- 25/arch/i386/mm/init.c~nx-update-2	2004-06-22 11:47:36.476784720 -0700
+++ 25-akpm/arch/i386/mm/init.c	2004-06-22 11:47:36.480784112 -0700
@@ -470,8 +470,6 @@ void __init paging_init(void)
 	set_nx();
 	if (use_nx)
 		printk("NX (Execute Disable) protection: active\n");
-	else
-		printk("NX (Execute Disable) protection: not present!\n");
 #endif
 
 	pagetable_init();
_
